frontend/app/components/mapbox/filter-features.js

```diff
@@ -0,0 +1,31 @@
+import Component from '@ember/component';
+import { computed } from '@ember-decorators/object';
+
+export default class MapboxFilterFeaturesComponent extends Component {
+  map = {}
+  layerId = null;
+  filterById = "id";
+  // map will be restricted to this array of features
+  featureIds = null;
+
+  @computed('featureIds', 'filterById')
+  get filterToFeatures(){
+    const { instance } = this.map;
+    if(this.featureIds && this.featureIds.length > 0 && (this.featureIds[0] != null)){
```

Comment:
I'm confused by the complexity here... seems unnecessary... we could set a default for `this.featureIds` to an empty array (or something that mapbox-gl expects). `this.featureIds.length` will be falsey if it's 0 anyways. so no need for the greater than. `(this.featureIds[0] != null)` is there really a scenario where we have an array of nulls? 

This seems like something upstream could be validated better.

Reply:
On load, `StudyAreaMap.hoveredFeatureId` is set to null. 
On the upcoming commit amend I did change FilterFeatures to just strip out all `null` values on filter construction, in anticipation of a list of feature Ids. 

I think this was the code is also being defensive against any tract geoids being null for some reason.